src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -977,12 +977,19 @@ private static SuggestedActionSetPriority GetSuggestedActionSetPriority(CodeActi
                     return null;
                 }
 
+                var sourceText = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+
+                var pastedTextSpanOpt =
+                    provider._pasteTrackingService.TryGetPastedTextSpan(sourceText.Container, out var pastedTextSpan)
+                    ? (TextSpan?)pastedTextSpan
+                    : null;
```

Comment:
```suggestion
                var pastedTextSpanOpt = provider._pasteTrackingService.TryGetPastedTextSpan(sourceText.Container, out var pastedTextSpan)
                    ? (TextSpan?)pastedTextSpan
                    : null;
```

The problem with this style is how difficult it is to distinguish the parent/child relationship.  For exaple, this would be even more difficult to understand:

```suggestion
                var pastedTextSpanOpt = provider._pasteTrackingService.TryGetPastedTextSpan(sourceText.Container, out var pastedTextSpan)
                    ? another.complex.expr()
                    ? and.yet.another()
                    ? baz
                    : quuz
                    : bar
                    : null;
```

As opposed to:

```suggestion
                var pastedTextSpanOpt = provider._pasteTrackingService.TryGetPastedTextSpan(sourceText.Container, out var pastedTextSpan)
                    ? another.complex.expr()
                        ? and.yet.another()
                            ? baz
                            : quuz
                        : bar
                    : null;
```

Now i can very easily see what children are actually siblings, and i can easily tell the parent/child relationship.

i *get* that you continually push this strict adherence to indentation.   But it's not going to fly with me.  Custom indentation serves a valuable purpose and we do judiciously use it across the IDE codebase where appropriate because it really helps with clarity.

Reply:
This is all covered by https://github.com/dotnet/roslyn/issues/38255#issuecomment-546023390. The form I used is correct (covered by 1(ii)), as are the first and third examples here (covered by 1(iii)).

Reply:
i don't actually see the value of 1ii.  in particular it seems to entirely not what i'd want for indenting whenever the expression itself starts on a new line.  i.e. on a new line you'll always get:

```
....
    startOfExpr
    ? whenTrue
    : whenFalse
```

Which never seems desirable.  This also doesn't really match teh examples shown in that linked post, none of which show the whenTrue/whenFalse parts aligning with the condition.